taxonomies/ingredients.txt

```diff
@@ -46917,6 +46922,8 @@ nl:Gepelde walnoot
 # openfoodfacts:https://world.openfoodfacts.org/ingredient/fr:cerneaux-de-noix
 # 78 products in 4 languages @2018-10-05
 
+<en:wallnut
```

Comment:
wallnut with 2 l's is a typo

Reply:
It's everywhere! You're right I'll correct it, obviously been there forever.